routes/instances.js

```diff
@@ -30,6 +30,15 @@ module.exports = {
 		getWorldometerPage,
```

Comment:
why return all of them if you now only need 1 function (executeScrape)

Reply:
Some of the functions are referenced in other places.

Reply:
good argument, not gonna lie